<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolManagerDialog</class>
 <widget class="QDialog" name="toolManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/toolManage.png</normaloff>:/icons/icons/toolManage.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="3">
    <widget class="QListView" name="loadedToolsListView"/>
   </item>
   <item row="6" column="1">
    <widget class="QToolButton" name="upButton">
     <property name="text">
      <string>up</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/up.png</normaloff>:/icons/icons/up.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QListView" name="avaliableToolsListView"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Loaded Tools:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Available Tools:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="downButton">
     <property name="text">
      <string>down</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/down.png</normaloff>:/icons/icons/down.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
